scene/gui/button.cpp

```diff
@@ -138,8 +139,6 @@ void Button::_notification(int p_what) {
 						}
 						if (has_theme_color(SNAME("font_hover_pressed_color"))) {
 							color = get_theme_color(SNAME("font_hover_pressed_color"));
-						} else {
-							color = get_theme_color(SNAME("font_color"));
```

Comment:
This removal is not really necessary, but then again, it's super-unlikely that the `else` would be ever executed.